JiraReporter/Program.cs

```diff
@@ -23,21 +23,24 @@ static void Main(string[] args)
             policy.SetCurrentOverride(options);
             policy.SetPermanentTaskLabel();
             policy.SetDraftMode(options);
+
             if (RunReport(policy, options) == true)
                 RunReportTool(args, policy, options);
+            else
+                Console.WriteLine("Unable to run report tool due to policy settings or final report already generated.");
         }
 
         private static bool RunReport(SourceControlLogReporter.Model.Policy policy, SourceControlLogReporter.Options options)
         {
-            var today = DateTime.Now.ToOriginalTimeZone().Date;
-            if (policy.IsDraft == false && today == policy.LastReportSentDate.Date)
-                return false;
-            if (policy.IsDraft == true && today == policy.LastReportSentDate.Date)
+            if (policy.LastReportSentDateFormat.Date.DayOfYear == DateTime.Now.ToOriginalTimeZone().Date.DayOfYear)
```

Comment:
@vladichimescu 
Nu imi dau seama aici de ce s-a schimbat verificarea anterioara. 
Concret de ce se foloseste DayOfYear. Nu cred ca era cazul sa folosesti day of year ci sa compari direct datele, la fel cum era si inainte. 

Dar intr-adevar conditia cu policy.IsDraft practic nu conta si nu trebuia pusa.